deploy/bin/build-push-edge-endpoint-image.sh

```diff
@@ -37,6 +30,7 @@ aws ecr get-login-password --region ${ECR_REGION} | docker login \
                   --username AWS \
                   --password-stdin  ${ECR_URL}
 
+# TODO: delete this!!!
```

Comment:
Q: When do we plan to come back and delete this? When we remove the setup-ee stuff?

Reply:
Yes, because the existing setup-ee.sh tests use it. I have a checklist of stuff to do when I do that sweep.